scripts/jenkins/groovy/benchmark.groovy

```diff
@@ -0,0 +1,63 @@
+def call(buildConfig, stageConfig) {
+
+  def H2O_3_HOME = 'h2o-3'
+
+  if (stageConfig.bucket == null) {
+    stageConfig.bucket = 'test.0xdata.com/h2o-3-benchmarks'
```

Comment:
Pls use the bucket which is use already for GBM benchmarks, better is to use less buckets, since we do not have good documentation for tracking them.

Furthermore, do not use test.0xdata.com bucket since it could be used as DNS entry.

Reply:
thanks, I'll change it. I'm using the test bucket because of WIP. If I change it to s3://test.... would it be ok?

Reply:
👍